source/stylesheets/card.scss

```diff
@@ -33,3 +29,52 @@ p.card__prompt {
   font-size: $base__font-size;
   line-height: $base-line-height;
 }
+
+.card__refresh-button {
+  background-color: $blue-fill;
+  border-radius: $base-border-radius;
+  border: $base-border;
+  color: $white;
+  overflow: hidden;
+  padding: .75rem 2rem;
+  position: relative;
+  text-decoration: none;
+  transition: all 0.2s linear 0s;
+
+  &:before {
+    align-items: center;
+    background-color: rgba( #000,0.3);
+    border-radius: 0 50% 50% 0;
+    color: $yellow;
+    content: "\2191";
+    display: inline-flex;
+    font-size: $base__font-size;
+    font-weight: $h1-font-weight;
+    height: 100%;
+    justify-content: center;
+    left: 0px;
+    position: absolute;
```

Comment:
If you're interested, it might be a fun exercise to add Bourbon in a later PR and see the kinds of mixins it could help with. For example, we could consolidate all of the positioning mixins into:

`@include position(absolute, 0 null null 0);`